themes/default/content/blog/announcing-aws-native/index.md

```diff
@@ -55,6 +59,110 @@ const objectlambda = new awsnative.s3objectlambda.AccessPoint("objectlambda-ap",
 });
 ```
 
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+import pulumi
+import pulumi_aws_native as aws_native
+
+my_bucket = aws_native.s3.Bucket("myBucket")
+ap = aws_native.s3.AccessPoint("ap", bucket=my_bucket.id)
+objectlambdaap = aws_native.s3objectlambda.AccessPoint("objectlambdaap", object_lambda_configuration=aws_native.s3objectlambda.AccessPointObjectLambdaConfigurationArgs(
+    supporting_access_point=ap.arn,
+    transformation_configurations=[aws_native.s3objectlambda.AccessPointTransformationConfigurationArgs(
+        actions=["GetObject"],
+        content_transformation={
+            "awsLambda": {
```

Comment:
I believe this and the `functionArn` need to be PascalCase?

Reply:
Sounds good. Note that `cf2pulumi` generated this code so either I mis-wrote my CloudFormation (very likely) or we may have a bug.

Reply:
I think there is likely a cf2pulumi bug in that is is overly-camelCasing these inputs.  Will confirm that in a bit and if so open an issue to track.